packages/interactive_media_ads/lib/src/android/interactive_media_ads_proxy.dart

```diff
@@ -0,0 +1,130 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'interactive_media_ads.g.dart';
+
+/// Handles constructing objects and calling static methods for the Android
+/// Interactive Media Ads native library.
+///
+/// This class provides dependency injection for the implementations of the
+/// platform interface classes. Improving the ease of unit testing and/or
+/// overriding the underlying Android classes.
+///
+/// By default each function calls the default constructor of the class it
+/// intends to return.
+class InteractiveMediaAdsProxy {
+  /// Constructs an [InteractiveMediaAdsProxy].
+  const InteractiveMediaAdsProxy({
+    this.newVideoProgressUpdate = VideoProgressUpdate.new,
+    this.newFrameLayout = FrameLayout.new,
+    this.newVideoView = VideoView.new,
+    this.newVideoAdPlayer = VideoAdPlayer.new,
+    this.newAdsLoadedListener = AdsLoadedListener.new,
+    this.newAdErrorListener = AdErrorListener.new,
+    this.newAdEventListener = AdEventListener.new,
+    this.createAdDisplayContainerImaSdkFactory =
+        ImaSdkFactory.createAdDisplayContainer,
+    this.instanceImaSdkFactory = _instanceImaSdkFactory,
+    this.videoTimeNotReadyVideoProgressUpdate =
+        _videoTimeNotReadyVideoProgressUpdate,
+  });
+
+  /// Constructs [VideoProgressUpdate].
+  final VideoProgressUpdate Function({
+    required int currentTimeMs,
+    required int durationMs,
+  }) newVideoProgressUpdate;
+
+  /// Constructs [FrameLayout].
+  final FrameLayout Function() newFrameLayout;
+
+  /// Constructs [VideoView].
+  final VideoView Function({
+    required void Function(
+      VideoView,
+      MediaPlayer,
+      int,
+      int,
+    ) onError,
+    void Function(
+      VideoView,
+      MediaPlayer,
```

Comment:
Wouldn't some of these be more readable without the trailing comma? Or would the inconsistency hurt readability?

Reply:
Yea, it does make it more readable. I removed the trailing commas